packages/web/src/components/Alert/_theme.scss

```
@@ -0,0 +1,19 @@
+@use 'sass:map';
+@use '@tokens' as tokens;
+
+$typography: tokens.$body-medium-text-regular;
+
+$success-color: tokens.$emotion-success-default;
+$success-background-color: tokens.$emotion-success-background;
+$danger-color: tokens.$emotion-danger-default;
+$danger-background-color: tokens.$emotion-danger-background;
+
+$border-radius: tokens.$radius-200;
+$space-inner: tokens.$space-700;
+$space-icon: tokens.$space-600;
+$icon-width: tokens.$space-700;
+
+$emphasis-font-weight: map.get(tokens.$body-small-text-bold, 'mobile', 'font-weight');
```

```suggestion
$emphasis-font-weight: map.get(tokens.$body-medium-text-bold, 'mobile', 'font-weight');
```
Why not use the same style.

---

I was to so sure. As I wrote earlier, there were two types of typography in Figma.

---

Fixed.

---

This should match the base style in `$typography`, shouldn't it? I mean when there is `$body-medium-text-regular` in `$typography`, then `$body-medium-text-bold` should be here.

---

Fixed.